buildSrc/build.gradle

```diff
@@ -0,0 +1,13 @@
+plugins {
+    id 'java'
+    id 'groovy'
+}
+
+sourceCompatibility = 1.8
+
+repositories {
+    mavenCentral()
+}
```

Comment:
Isn't this now duplicated?

Reply:
This specifies the repositories used for resolving dependencies for the buildSrc project.
As far as I know, this does not overwrite or duplicate the `buildscript.repositories` property in the `build.gradle`.